source/georef/georef.h

```diff
@@ -155,7 +156,7 @@ struct Way :public nt::Nameable, nt::Header{
     nt::GeographicalCoord projected_centroid(const Graph&) const;
     nt::MultiLineString make_multiline(const Graph&) const;
     template<class Archive> void serialize(Archive & ar, const unsigned int) {
-      ar & idx & name & comment & uri & way_type & admin_list & house_number_left & house_number_right & edges & geoms;
+      ar & idx & name & comment & uri & way_type & visible & admin_list & house_number_left & house_number_right & edges & geoms;
```

Comment:
This is why the version number is incremented

Reply:
Since the filtering was moved, we don't need to keep `visible` field in data.
Now, we simply don't load ways with `visible=false` into autocomplete data.

https://github.com/CanalTP/navitia/pull/2592/files#diff-992196e00edd443955062e4bad390541R420

Reply:
The version number no longer needs to be incremented